sdk/storage/azure-storage-queue/apireview/azure/storage/queue/aio/__init__.pyi

```diff
@@ -0,0 +1,8 @@
+# Stubs for azure.storage.queue.aio (Python 3)
+#
+# NOTE: This dynamically typed stub was automatically generated by stubgen.
+
+from ..models import AccessPolicy as AccessPolicy, CorsRule as CorsRule, Logging as Logging, Metrics as Metrics, QueueMessage as QueueMessage, QueuePermissions as QueuePermissions, QueueProperties as QueueProperties, RetentionPolicy as RetentionPolicy
+from .models import MessagesPaged as MessagesPaged, QueuePropertiesPaged as QueuePropertiesPaged
```

Comment:
If we believe the return type should just says "async iterator", then these types should not be exposed.

Reply:
#7863